Postgres/AddServer.swift

```diff
@@ -13,17 +13,29 @@ class AddServerViewController: NSViewController, MainWindowModelConsumer {
 	@objc dynamic var mainWindowModel: MainWindowModel!
 	@objc dynamic var name: String = "New Server"
 	@objc dynamic var port: UInt = 5432
+    @objc dynamic var binPath: String = ""
+    @objc dynamic var binPathEditable: Bool = false
```

Comment:
An alternative might be to hide the "Binaries" field instead of disabling it when a built-in version is selected. That would reduce the complexity of the form for 99% of users who don't care where their binaries are.